extensions/transactions/simple/examples/json/ex02_replace.adoc

```diff
@@ -0,0 +1,37 @@
+....
+   Client                                                              Server
+     |                                                                   |
+     | PUT /collections/oakland_buildings/items/OB.2   HTTP/1.1          |
+     | Content-Type: application/geo+json                                |
+     |                                                                   |
+     | {                                                                 |
+     |   "type": "Feature",                                              |
+     |   "id": "OAKLAND_BUILDINGS.2",                                    |
```

Comment:
The `id` should be `OB.2` (or the `{featureId} ` in the PUT request should be `OAKLAND_BUILDINGS.2`)